### PR TITLE
Fix ovirt-image-template parameters

### PR DESCRIPTION
**Pull request by machacekondra**
Related-to: https://bugzilla.redhat.com/show_bug.cgi?id=1489454

### CODE REVIEW

File: roles/ovirt-image-template/tasks/qcow2_image.yml [hunk 1]
```diff
@@ -62,9 +62,9 @@
     ovirt_disks:
       auth: "{{ ovirt_auth }}"
       name: "{{ template_disk_name | default(template_name) }}"
-      interface: "{{ template_disk_format | default('virtio') }}"
-      size: "{{ template_disk_size}}"
-      format: "{{  template_disk_interface | default('cow') }}"
+      interface: "{{ template_disk_interface }}"
+      size: "{{ template_disk_size }}"
+      format: "{{  template_disk_format }}"
```
Comment (mwperina) [inline roles/ovirt-image-template/tasks/qcow2_image.yml]: Default values are no longer needed?

Comment (mwperina) [reply roles/ovirt-image-template/tasks/qcow2_image.yml]: Ahh, please ignore, just saw them defaults/main.yml

**Review by mwperina — APPROVED** [review]
👍

**Review by tareqalayan — APPROVED** [review]

**Comment by machacekondra** [comment]
@mwperina As we don't support RAW disk images, I've removed the `template_disk_format` variable.

**Comment by machacekondra** [comment]
@mwperina Actually we may leave it there, but if user will use `raw` with `qcow` image it will fail. But in future we may add support to upload also `raw` images, not only `qcow2`, so we can leave the variable there, but if user will use it wrongly, it may fail for them..

**Pull request merged**